src/entities/editEntries.js

```diff
@@ -165,9 +166,15 @@ export const createEntry = ({
     hasError = false,
     color,
 }) => {
+    const {
+        imageDetails,
+    } = data;
+
     const keysToPick = [
         'excerpt',
         'image',
+        'imageRaw',
+        'leadImage',
```

Comment:
This seems weird.
1. What is leadImage on entries? Where is this used?
2. What is the difference between image and imageRaw? Do we need both?

Reply:
1. LeadImage is the `id` of image coming from lead if we are dragging from the `images column.
2. ImageRaw is the blob we save before creating of image file
3. Image is the generated file URL